portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java

```diff
@@ -36,7 +44,32 @@
 public class NewCommentServlet extends HttpServlet {
 
   public static final String MAIN_PAGE_URL = "/";
+
+  // Get the comment sentiment using Google's Sentiment Analysis API
+  public float getCommentSentiment(String comment) throws IOException{
+    Document sentimentDoc =
+      Document.newBuilder().setContent(comment).setType(Document.Type.PLAIN_TEXT).build();
+    LanguageServiceClient languageService = LanguageServiceClient.create();
+    Sentiment sentiment = languageService.analyzeSentiment(sentimentDoc).getDocumentSentiment();
+    float sentimentScore = sentiment.getScore();
+
+    languageService.close();
+
+    return sentimentScore;
+  }
   
+  /** Starts a simulated HashMap that will store the comment translations */
+  public EmbeddedEntity addCodeLanguage(String comment) throws IOException {
```

Comment:
```suggestion
  public EmbeddedEntity addTranslation (String comment) throws IOException {
```
The name of this function is a little confusing. Are you adding translations for the comment? If so, maybe consider a more specific name.

Reply:
Indeed it is! The idea is to start an Embedded Entity (which can be seen as a Hash map) which will hold the translations of the comment in every language requested. Surely can have that more descriptive name, thanks for the suggestion Aki!